lib/paquito/active_record_coder.rb

```diff
@@ -99,18 +102,25 @@ def attributes_for_database(record)
         attributes
       end
 
-      def deserialize_record(class_name, attributes_from_database, new_record = false)
```

Comment:
Hmm, this may encounter problems on deploy if instances with the new version of the gem is serializing with four elements, but instances with the old version are loading back and passing to this version of the method, which would then fail.

This needs to be a breaking change with a transition step where we change this method to accept any number of arguments and just throw away the ones that aren't used.